xclim/sdba/adjustment.py

```diff
@@ -370,6 +374,7 @@ def _train(
         kind: str = ADDITIVE,
         group: str | Grouper = "time",
         adapt_freq_thresh: str | None = None,
+        jitter_under_thresh_value: str | None = None,
```

Comment:
not a fan of this keyword jitter_under_thresh_value, but there is already a function jitter_under_thresh, so not sure it's a good idea to use the same name for a string input